libr/core/cmd_anal.c

```diff
@@ -7717,17 +7723,43 @@ static void cmd_anal_graph(RCore *core, const char *input) {
 		break;
 	case 'd': // "agd"
 		switch (input[1]) {
-		case 'v':
-		case 't':
-		case 'j':
-		case 'J':
-		case 'g':
-		case 'k':
-		case '*':
+		case 'j': {
```

Comment:
calculate the common part of the mask before `R_CORE_ANAL_GRAPHBODY | R_CORE_ANAL_GRAPHDIFF`, since all of these share it

Reply:
Done